### PR TITLE
Add palindrome check of numbers

### PR DESCRIPTION
**Pull request by sandeepN06**
Palindrome check of String along with numbers included in the code snippet

### CODE REVIEW

File: python/palindrome_checker.py [hunk 1]
```diff
@@ -12,3 +14,17 @@ def palindrome (s):
 s = input()
```
Comment (tasnimzotder) [inline python/palindrome_checker.py]: Take the input as `Enter a string: `.
and the output should be like `The string is not palindrome`

Comment (sandeepN06) [reply python/palindrome_checker.py]: Sure assign it to me , i am thankful to edit this

Comment (tasnimzotder) [reply python/palindrome_checker.py]: Sure

**Review by tasnimzotder — APPROVED** [review]
👍

**Comment by sandeepN06** [comment]
Was your preffered issue cleared? :)

**Pull request merged**